Formula/gst-devtools.rb

```diff
@@ -0,0 +1,40 @@
+class GstDevtools < Formula
+  include Language::Python::Shebang
+
+  desc "GStreamer development and validation tools"
+  homepage "https://gstreamer.freedesktop.org/modules/gstreamer.html"
+  url "https://gstreamer.freedesktop.org/src/gst-devtools/gst-devtools-1.18.0.tar.xz"
+  sha256 "82337141b5654f11c440f783892ba9d9498b3b6b98c2286b000f96dce6945f16"
+  license "LGPL-2.1-or-later"
+  head "https://anongit.freedesktop.org/git/gstreamer/gst-devtools.git"
+
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gettext"
+  depends_on "gst-plugins-base"
+  depends_on "gstreamer"
+  depends_on "json-glib"
+  depends_on "python@3.8"
+
+  def install
+    args = std_meson_args + %w[
+      -Dintrospection=enabled
+      -Dvalidate=enabled
+      -Dtests=disabled
+    ]
+
+    mkdir "build" do
+      system "meson", *args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+
+    rewrite_shebang detected_python_shebang, bin/"gst-validate-launcher"
+  end
+
+  test do
+    system "#{bin}/gst-validate-launcher", "--usage"
```

Comment:
Can we get a better test than this? This is similar to a `--help` test right?

Reply:
I copied the test block from `gst-validate`... honestly I have no idea what this package does, I have never used GStreamer 😄

Reply:
Time to find out 😄

Reply:
Sorry, but I don't have that kind of time right now. 

Would be grateful if you could merge it in regardless.